File: Server/GridiaServer/src/main/java/hoten/gridiaserver/Sector.java
package hoten.gridiaserver;

public class Sector {
    private final int _x, _y, _z;
    public final Tile[][] _tiles;
    
    public Sector(Tile[][] tiles, int x, int y, int z) {
        _tiles = tiles;
        _x = x;
        _y = y;
        _z = z;
    }
    
    public Tile getTile(int x, int y) {
        return _tiles[x][y];
    }
}

File: Client/Assets/Scripts/Sector.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Gridia
{
    public class Sector
    {
        private Tile[,] _tiles;

        public Sector(Tile[,] tiles) {
            _tiles = tiles;
        }

        public Tile GetTile(int x, int y)
        {
            return _tiles[x, y];
        }
    }
}
